iac-extensions/arm/src/main/java/org/sonar/iac/arm/tree/impl/json/ObjectExpressionImpl.java

```diff
@@ -42,6 +43,11 @@ public List<Property> properties() {
     return Collections.unmodifiableList(properties);
   }
 
+  @Override
+  public List<ResourceDeclaration> nestedResources() {
+    return Collections.emptyList();
```

Comment:
Why it returns empty list? I would expect some logic here to return nested resources 🤔